lib/pages/launch_page.dart

```diff
@@ -1 +1,41 @@
-//
\ No newline at end of file
+import 'package:flutter/material.dart';
+
+import 'search_page.dart';
+
+class LaunchPage extends StatefulWidget {
+  const LaunchPage({super.key, required this.title});
+
+  final String title;
+
+  @override
+  State<LaunchPage> createState() => _LaunchPageState();
+}
+
+class _LaunchPageState extends State<LaunchPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            OutlinedButton(
+                onPressed: () {
+                  Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                          builder: (context) => const SearchPage(title: 'Search',)));
```

Comment:
```suggestion
                          builder: (context) => const SelectPage(title: 'SelectPage',)));
```
遷移先は、LaunchPage -> SelectPage -> SearchPage であってますでしょうかー？？

Reply:
この段階ではそうなってますね。　ただし、今ローカルで動かして動画にしている段階では、
main →　launch →　select（ランダムか絞り込みか選ぶ）→　ここでアニメーション と絞り込みに分岐です。

Reply:
承知です！ありがとうございます!